src/api/app/views/webui/users/index.html.haml

```diff
@@ -18,9 +18,14 @@
       %tbody
 
     .pt-4
-      = link_to(new_user_path(pagetitle: 'create user', submit_btn_text: 'create')) do
-        %i.fas.fa-plus-circle.text-primary
-        Create User
+      - if feature_enabled?(:responsive_ux)
+        - content_for :actions do
+          %li.nav-item
```

Comment:
I think line should be removed, right?

Reply:
Yes, you are right!

Reply:
I went with not using the helper method as it introduced some dependencies in the rest of branches I'm working on. This is simpler